samples/wifi/shell/boards/nrf7002dk_nrf5340_cpuapp_ns.conf

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Store Wi-Fi credentials using PSA storage
+CONFIG_WIFI_CREDENTIALS_BACKEND_PSA=y
```

Comment:
Should we also change `WIFI_CREDENTIALS_BACKEND` default value if building for NS `if TF-M` or equivalent?

Reply:
Sounds like a good idea

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/12793 takes care of this, please have a look.

Reply:
Sure, just approved that PR, thanks.